### PR TITLE
[PLAY-330]-Lightbox Kit Responsive on Mobile

### PR DESCRIPTION
**Pull request by nidaqg**
#### Screens

Screen Shot 2022-09-23 at 1 32 55 PM

#### Breaking Changes

No, fixes bug where thumbnails at bottom of screen were not visible on chrome and safari on mobile + adds swiping functionality to slides on mobile

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-330)

#### How to test this:
- On Browser: Test Lightbox kit 'Multiple', make sure thumbnails on bottom of screen scroll + active thumbnail stays centered on screen
- On Mobile: Test Lightbox kit 'Multiple', make sure thumbnails show up on bottom of screen, are scrollable and active thumbnail stays centered on screen. Test to make sure slides are swipeable.

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_lightbox/lightbox.scss [hunk 1]
```diff
@@ -3,56 +3,7 @@
...
-  .image-gallery-thumbnail-image {
-    height: 64px;
-    width: 93px;
-  }
-
-  .image-gallery-thumbnail + .image-gallery-thumbnail {
-    margin-left: 2px;
-    margin-top: 2px;
-  }
+ .carousel {
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_lightbox/lightbox.scss:6]: Thank you for cleaning up as you went! Great job! @nidaqg

File: playbook/app/pb_kits/playbook/pb_lightbox/lightbox.scss [hunk 2]
```diff
@@ -217,6 +163,8 @@ $slides-margin: $space-md;
 
 .Thumbnail.active {
   padding: 6px;
+  display: flex;
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_lightbox/lightbox.scss:166]: Easy peasy!

Comment (nidaqg) [reply playbook/app/pb_kits/playbook/pb_lightbox/lightbox.scss:166]: Always easy in retrospect 😂

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_lightbox/lightbox.scss:166]: Very true!

**Review by thestephenmarshall — APPROVED** [review]

**Pull request merged**